src/osd/osd_types.h

```diff
@@ -5624,6 +5624,35 @@ struct watch_item_t {
     }
     DECODE_FINISH(bl);
   }
+  void dump_bare(ceph::Formatter *f) const {
+    f->dump_stream("watcher") << name;
+    f->dump_int("cookie", cookie);
+    f->dump_int("timeout", timeout_seconds);
+    f->open_object_section("addr");
+    addr.dump(f);
+    f->close_section();
+  }
+  void dump(ceph::Formatter *f) const {
+    f->open_object_section("watch_item_t");
+    dump_bare(f);
+    f->close_section();
```

Comment:
(for better or worse) the convention for dump() is to not include the enclosing object section